apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations-v2/[evaluationUuid]/editor/_components/EvaluationEditor/index.tsx

```diff
@@ -163,7 +184,8 @@ export function EvaluationEditor({
               />
               <TextEditor
                 compileErrors={metadata?.errors}
-                value={originalPrompt}
+                defaultValue={originalPrompt}
+                value={value}
```

Comment:
This was causing the lag. `value` is the local value that has to be passed to the editor to feel responsive